packages/schemas/src/lib/auth/auth-schemas.ts

```diff
@@ -64,3 +65,34 @@ export const JsonSignCustomSessionKeyRequestForPkpReturnSchema = z
       }),
     ])
   );
+
+/**
+ * Consolidated schema for PKP mint requests.
+ * This replaces the duplicated schemas across the codebase.
+ * Handles both string and number authMethodType inputs.
+ */
+export const MintPKPRequestSchema = z
+  .object({
+    authMethodId: HexPrefixedSchema,
+    authMethodType: z.coerce.number().pipe(z.nativeEnum(AUTH_METHOD_TYPE)),
+    pubkey: HexPrefixedSchema.default('0x'),
+    scopes: z.array(ScopeStringSchema).optional().default([]),
```

Comment:
```suggestion
    scopes: z.array(ScopeSchemaRaw).optional().default([]),
```
properly transform scope strings like "sign-anything" to their BigInt values (like 1n).